analyses/molecular-subtyping-HGG/02-HGG-molecular-subtyping-subset-files.R

```diff
@@ -88,7 +88,11 @@ path_dx_df <- tumor_metadata_df %>%
   # Inclusion on the basis of CBTTC harmonized pathology diagnoses
   filter(pathology_diagnosis %in% path_dx_list$exact_path_dx |
          # Inclusion based on pathology free text diagnosis
-         pathology_free_text_diagnosis ==path_dx_list$gliomatosis_path_free_text_exact)
+         pathology_free_text_diagnosis ==path_dx_list$gliomatosis_path_free_text_exact |
```

Comment:
```suggestion
         pathology_free_text_diagnosis == path_dx_list$gliomatosis_path_free_text_exact |
```
space. I know that is from before, but we can update here!